### PR TITLE
graph visualisation of system initialization process

### PR DESCRIPTION
**Pull request by dyniec**
Requires `graphviz` package.
Usage (in main mimiker directory):
`python3 script/show-sysinit.py | dot -Tpng > a.png`.

### CODE REVIEW

File: script/show-sysinit.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    command='grep {}/* -e SYSINIT_ADD'.format(folder)
+    out=subprocess.check_output(command,shell=True)
+    out=out.decode('UTF-8')
+    for line in out.split('\n'):
+      if not 'SYSINIT_ADD' in line:
+        continue
+      line=line.partition('(')[2]
+      line=line.rpartition(')')[0]
+      name,fname,deps=line.split(', ',2)
+      # print('{} [label={}];'.format(name,fname))
```
Comment (czapiga) [inline script/show-sysinit.py]: Remove comment

File: script/show-sysinit.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+if __name__ == "__main__":
+  print('digraph mimiker_modules {')
+  for folder in ['sys','mips']:
+    command='grep {}/* -e SYSINIT_ADD'.format(folder)
+    out=subprocess.check_output(command,shell=True)
+    out=out.decode('UTF-8')
+    for line in out.split('\n'):
+      if not 'SYSINIT_ADD' in line:
+        continue
+      line=line.partition('(')[2]
```
Comment (cahirwpz) [inline script/show-sysinit.py]: This looks kinda verbose, I guess you could make better use of Python's regular expressions...

File: script/show-sysinit.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import os.path
+import sys
+import subprocess
+
+if __name__ == "__main__":
+  print('digraph mimiker_modules {')
+  for folder in ['sys','mips']:
```
Comment (cahirwpz) [inline script/show-sysinit.py]: Python has lots of great tools for processing text, I guess you could rewrite the above to something similar to:
```python
sysinit = re.compile('SYSINIT_ADD')
for source in map(os.listdir, ['sys', 'mips']):
  lines = filter(sysinit.search, file(source).readlines())
```

**Comment by cahirwpz** [comment]
Please fix all errors and warnings reported by `pep8`. The script is installed on mimiker server:
```
# pep8 script/show-sysinit.py
script/show-sysinit.py:3:10: E401 multiple imports on one line
script/show-sysinit.py:5:1: E302 expected 2 blank lines, found 1
...
```

Just a side note... probably `pep8` check should be integrated into our build process.

**Comment by cahirwpz** [comment]
Please implement #345, so that code style problem is solved once and for all, and return here.

**Pull request merged**